Gemfile.lock

```diff
@@ -1,94 +1,94 @@
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       mail (>= 2.7.1)
-    actionmailer (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     aws-eventstream (1.1.0)
-    aws-partitions (1.353.0)
-    aws-sdk-core (3.104.3)
+    aws-partitions (1.399.0)
+    aws-sdk-core (3.109.3)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.36.0)
-      aws-sdk-core (~> 3, >= 3.99.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.76.0)
-      aws-sdk-core (~> 3, >= 3.104.1)
+    aws-sdk-s3 (1.85.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.1)
+    aws-sigv4 (1.2.2)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.15)
-    better_errors (2.7.1)
+    bcrypt (3.1.16)
```

Comment:
### [bcrypt](https://github.com/codahale/bcrypt-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.1.15 → 3.1.16

[_~~change-log~~, [source-code](https://github.com/codahale/bcrypt-ruby)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/codahale/bcrypt-ruby/compare/v3.1.15...v3.1.16).

These are the individual commits:
- (8f99fc1) [Add ruby 2.7 to the test matrix](https://github.com/codahale/bcrypt-ruby/commit/8f99fc1cc31b51269fa951708ba3a62ceb0cc8e4)
- (7460f0b) [Install a recent bundler version](https://github.com/codahale/bcrypt-ruby/commit/7460f0b57b2f93862001ce9d2c4df1c94bd67697)
- (cedc403) [Remove Gemfile.lock](https://github.com/codahale/bcrypt-ruby/commit/cedc403dc61f621b9ab678337f9f68f784a45439)
- (9f3fb05) [Merge pull request #231 from agrare/add_ruby_2_7_to_test_mat…](https://github.com/codahale/bcrypt-ruby/commit/9f3fb05ba7b3732a87e0305ec635d7773bf1fafb)
- (77fa85d) [Fix crypt_r definition](https://github.com/codahale/bcrypt-ruby/commit/77fa85d25536822543253da63b0bfddf9d41ef57)
- (32469cc) [Merge pull request #234 from michelboaventura/fix_crypt_r](https://github.com/codahale/bcrypt-ruby/commit/32469cc1f220b6af2b1c663bb8d8d9f4e4249292)
- (abff57d) [bump version / fix changelog](https://github.com/codahale/bcrypt-ruby/commit/abff57ddc6ab6cbdadb486202c2942bfc3025ea6)


</details>